stdlib/crypto/src/main/ballerina/src/crypto/crypto.bal

```diff
@@ -218,121 +229,132 @@ function externCrc32b(byte[] input) returns handle = @java:Method {
-# + return - The generated signature or `Error` if private key is invalid
+# + return - The generated signature or `crypto:Error` if private key is invalid
 public function signRsaSha512(byte[] input, PrivateKey privateKey) returns byte[]|Error = @java:Method {
     name: "signRsaSha512",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-MD5 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaMd5Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaMd5Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                       returns boolean|Error = @java:Method {
     name: "verifyRsaMd5Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA1 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha1Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaSha1Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                        returns boolean|Error = @java:Method {
     name: "verifyRsaSha1Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA256 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha256Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
 public function verifyRsaSha256Signature(byte[] data, byte[] signature, PublicKey publicKey)
                                          returns boolean|Error = @java:Method {
     name: "verifyRsaSha256Signature",
     class: "org.ballerinalang.stdlib.crypto.nativeimpl.Sign"
 } external;
 
 # Verify RSA-SHA384 based signature.
+# ```ballerina boolean|crypto:Error validity = crypto:verifyRsaSha384Signature(data, signature, publicKey); ```
 #
 # + data - The content to be verified
 # + signature - Signature value
 # + publicKey - Public key used for verification
-# + return - Validity of the signature or `Error` if public key is invalid
+# + return - Validity of the signature or `crypto:Error` if public key is invalid
```

Comment:
```suggestion
# + return - Validity of the signature or `crypto:Error` if public key is invalid
```
Please refer the template for the correct format.

Reply:
Corrected.